modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js

```diff
@@ -84,6 +85,10 @@ const CreationMenu = ({
 		getVisibleItemsCount()
 	);
 
+	const hasIcons = () => {
+		return primaryItems.some((item) => item.icon);
```

Comment:
Nice! Minor nit: this can be inline, no need for `hasIcons`. Non-blocking.

Reply:
:+1: Done in [https://github.com/liferay-frontend/liferay-portal/pull/805/commits/bc125ba882fe46e9c4cf321106c6ffc48d474e8c](https://github.com/liferay-frontend/liferay-portal/pull/805/commits/bc125ba882fe46e9c4cf321106c6ffc48d474e8c)